### DISCUSSION TITLE
Is there a demo link for dashboard template?

### DISCUSSION THREAD

**Discussion by cenksener** [saadeghi/daisyui#2563] [General]
https://daisyui.lemonsqueezy.com/ i want to buy this template. But firstly i want to try it in mobile browser. Is there demo link?

**Comment by saadeghi** [comment]
I can't share a live demo but there are more pictures in the checkout page:  
https://daisyui.lemonsqueezy.com/checkout/buy/3c805f3d-77a5-440b-9e4d-e775005db6c9

And here's the fullpage screenshot in mobile screen:  

![localhost_5173_ (8)](https://github.com/saadeghi/daisyui/assets/7342023/017e6fa1-8eff-4ef8-a22a-fe4249d61f94)


Let me know if you have a question.
